src/java/test/org/apache/zookeeper/server/quorum/QuorumPeerConfigTest.java

```diff
@@ -103,6 +103,23 @@ public void testCustomSSLAuth()
         }
     }
 
+    /**
+     * Test case for https://issues.apache.org/jira/browse/ZOOKEEPER-2873
+     */
+    @Test
+    public void testSamePortConfiguredForClientAndElection() throws IOException, ConfigException {
```

Comment:
nit - ConfigException is unused

Reply:
Thanks, I will fix this with an amend commit (I don't want a whole commit in history just for this change)